changelog.md

```diff
@@ -1,3 +1,12 @@
+## <a name="2.10.0"/> 2.10.0 ##
+
+* Add support for creating geometry collections
```

Comment:
Is it not possible before? There are some existing types related to geometry what are they for

Reply:
Geometry collections support is added only now. Spatial types are common to both Geometry and Geography. Geography collections were supported from the beginning, so types have been around too.